src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -508,5 +512,51 @@ protected override bool TryExec(Guid commandGroup, uint commandId)
 
             return false;
         }
+
+        private IStreamingFindUsagesPresenter GetStreamingPresenter()
+        {
+            try
+            {
+                return _streamingPresenters.FirstOrDefault()?.Value;
+            }
+            catch
+            {
+                return null;
+            }
+        }
+
+        private async void FindReferences(
```

Comment:
❕  Missing `Async` suffix.
❕  Don't use `async void`. Return a `Task` even if you don't intend to wait on it specifically.